Packs/FeedLOLBAS/Playbooks/playbook-Compare_Process_Execution_Arguments_To_LOLBAS_Patterns.yml

```diff
@@ -0,0 +1,393 @@
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: c61b97f4-f754-44fa-882b-796c38ed1b78
+    type: condition
+    task:
+      id: c61b97f4-f754-44fa-882b-796c38ed1b78
+      version: -1
+      name: Has results?
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#default#':
+      - "4"
+      "yes":
+      - "13"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: foundIndicators.commands
+                accessor: command
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 790,
+          "y": 340
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: f0c62078-cd49-4a32-83f2-7e1db923d75f
+    type: condition
+    task:
+      id: f0c62078-cd49-4a32-83f2-7e1db923d75f
+      version: -1
+      name: Does input data exist?
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#default#':
+      - "4"
+      "yes":
+      - "12"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: inputs.ProcessName
+            iscontext: true
+          right:
+            value: {}
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: inputs.Commandline
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 460,
+          "y": 0
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: 42c3975a-5d20-4292-8318-8798842917a0
+    type: title
+    task:
+      id: 42c3975a-5d20-4292-8318-8798842917a0
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 460,
+          "y": 1010
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: fa54f32d-9f9d-48dc-8308-561d639a9444
+    type: condition
+    task:
+      id: fa54f32d-9f9d-48dc-8308-561d639a9444
+      version: -1
+      name: Has results?
```

Comment:
already saw this task name, what results? (I know they are different, please specify)